htdocs/comm/propal/card.php

```diff
@@ -2753,53 +2753,50 @@
-	print '</tr>';
-
-	// Amount VAT
-	print '<tr><td>'.$langs->trans('AmountVAT').'</td>';
-	print '<td class="nowrap right amountcard">'.price($object->total_tva, '', $langs, 0, - 1, - 1, $conf->currency).'</td>';
+	print '';
+	print '<td class="titlefieldmiddle">' . $langs->trans('AmountVAT') . '</td>';
+	print '<td class="nowrap right amountcard">' . price($object->total_tva, '', $langs, 0, -1, -1, $conf->currency);
+	if (isModEnabled("multicurrency") && ($object->multicurrency_code && $object->multicurrency_code != $conf->currency)) {
+		print ' - ' . price($object->multicurrency_total_tva, '', $langs, 0, -1, -1, $object->multicurrency_code);
+	}
+	print '</td>';
 	print '</tr>';
 
-	// Amount Local Taxes
-	if ($mysoc->localtax1_assuj == "1" || $object->total_localtax1 != 0) { 	// Localtax1
-		print '<tr><td>'.$langs->transcountry("AmountLT1", $mysoc->country_code).'</td>';
-		print '<td class="nowrap right amountcard">'.price($object->total_localtax1, '', $langs, 0, - 1, - 1, $conf->currency).'</td>';
-		print '</tr>';
-	}
-	if ($mysoc->localtax2_assuj == "1" || $object->total_localtax2 != 0) { 	// Localtax2
-		print '<tr><td>'.$langs->transcountry("AmountLT2", $mysoc->country_code).'</td>';
-		print '<td class="nowrap right amountcard">'.price($object->total_localtax2, '', $langs, 0, - 1, - 1, $conf->currency).'</td>';
-		print '</tr>';
+	if ($mysoc->localtax1_assuj == "1" || $object->total_localtax1 != 0) {
+		print '<tr>';
+		print '<td class="titlefieldmiddle">' . $langs->transcountry("AmountLT1", $mysoc->country_code) . '</td>';
+		print '<td class="nowrap  amountcard">' . price($object->total_localtax1, '', $langs, 0, -1, -1, $conf->currency);
+		if (isModEnabled("multicurrency") && ($object->multicurrency_code && $object->multicurrency_code != $conf->currency)) {
+			print ' - ' . price($object->total_localtax1, '', $langs, 0, -1, -1, $object->multicurrency_code);
+		}
+		print '</td>';
+		print '';
+		if ($mysoc->localtax2_assuj == "1" || $object->total_localtax2 != 0) {
+			print '';
+			print '<td>' . $langs->transcountry("AmountLT2", $mysoc->country_code) . '</td>';
+			print '<td class="nowrap right amountcard">' . price($object->total_localtax2, '', $langs, 0, -1, -1, $conf->currency);
+			if (isModEnabled("multicurrency") && ($object->multicurrency_code && $object->multicurrency_code != $conf->currency)) {
+				print ' - ' . price($object->total_localtax2, '', $langs, 0, -1, -1, $object->multicurrency_code);
+			}
+			print '</td>';
+			print '</tr>';
+		}
 	}
 
-	// Amount TTC
-	print '<tr><td>'.$langs->trans('AmountTTC').'</td>';
-	print '<td class="nowrap right amountcard">'.price($object->total_ttc, '', $langs, 0, - 1, - 1, $conf->currency).'</td>';
+	print '<tr>';
+	print '<td>' . $langs->trans('AmountTTC') . '</td>';
+	print '<td class="nowrap amountcard">' . price($object->total_ttc, '', $langs, 0, -1, -1, $conf->currency) . '</td>';
```

Comment:
The complementary amount in foreign currency is missing for the AmountTTC.
We should have
`if (isModEnabled("multicurrency") && ($object->multicurrency_code && $object->multicurrency_code != $conf->currency)) {`
with the print of price($object->multicurrency_total_ttc...

Also instead of a separation between the 2 amount with a - , can you instead add a third column `<td></td>`